CHANGELOG.md

```diff
@@ -9,6 +9,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+### Changed
+
+### Fixed
+
+[Unreleased]: https://github.com/SumoLogic/sumologic-kubernetes-collection/compare/v2.4.0-rc.0...main
+
+## [v2.4.0_rc.0][v2_4.0_rc.0]
+
+### Release v2.4.0-rc.0 - 2022-01-20
```

Comment:
I somehow inclined to say:

```suggestion
### Released 2022-01-20
```

since the release is already stated in the section right above this line

Reply:
Good idea, I've changed that ;)